code/modules/organs/blood.dm

```diff
@@ -256,11 +271,11 @@ var/const/CE_STABLE_THRESHOLD = 0.5
 	if(!should_have_organ(O_HEART))
 		return null
 
-	if(vessel.get_reagent_amount("blood") < amount)
+	if(vessel.get_reagent_amount("blood") < amount || vessel.get_reagent_amount("blood") < BLOOD_MINIMUM_STOP_PROCESS)
```

Comment:
vessel.get_reagent_amount("blood") < amount || vessel.get_reagent_amount("blood") < BLOOD_MINIMUM_STOP_PROCESS

Could be replaced with

vessel.get_reagent_amount("blood") < max(amount, BLOOD_MINIMUM_STOP_PROCESS)